### PR TITLE
Landing page

### LINKED ISSUE #165: Improve Landing page UX
Issues to fix:
- hero section - CTAs should be more clear
- find a way to make it very clear which counties we are live in
- information on eligibility
- partner seals
- break up light blue section so there is not big chunks of content on the page
- see notes: https://docs.google.com/document/d/1wqOhP5joktWIobf8qsdBBe7j3jTXXybciGN2C0bXf9E/edit

### LINKED ISSUE #613: Write Partnerships page HTML using CfA styleguide
Mocks: https://drive.google.com/drive/u/0/folders/0B3BLnlbsdQ_hdU5JRWZlcVVEWFk

### PR DESCRIPTION
**Pull request by bengolder**
Closes #165
Closes #613 

**UAT:**
- [x] make sure that the [partnerships page](https://cmr-develop-pr-838.herokuapp.com/partnerships/) matches expectations.
    - [x] @jazmynlatimer
    - [x] @jennifermarie 
- [x] make sure that the [landing page](https://cmr-develop-pr-914.herokuapp.com/) matches expectations.
    - [x] @jazmynlatimer
    - [x] @jennifermarie 

Check that styles appear consistently across browsers:
  - [x] IE 9, Windows 7 Desktop
  - [x] Nokia Lumia 630 with Windows Phone 8.1
  - [x] Samsung Galaxy Note 3

Styleguide: https://getcalfresh.org/styleguide
Mocks: https://drive.google.com/drive/u/0/folders/0B3BLnlbsdQ_hdU5JRWZlcVVEWFk

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,6 +111,22 @@ db.pull.demo:
...
\ No newline at end of file
+static.dev:
+	sass \
+        --require bourbon \
+        --require normalize-scss \
+        --require neat \
+        --watch intake/static/intake/scss:intake/static/intake/css
+
+
+static.rebuild:
```
Comment (jennifermarie) [inline Makefile]: Slated for removal.

File: templates/main_splash.jinja [hunk 2]
```diff
@@ -1,237 +1,133 @@
...
+    </section>
 
-  </div>
-</section>
+  </div> {# end of page wrapper #}
 
+  {% include "gcf-style-footer.jinja" %}
 
-{% endblock content %}
+</body>
```
Comment (jennifermarie) [inline templates/main_splash.jinja]: Double-check that we are not duplicating close body tag.

File: intake/static/intake/scss/atoms/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.slab--not-padded {
+  padding: {
+    top: 0;
+    bottom: 0;
+  }
+}
+
+header.main-header + .slab--header {
```
Comment (norrishung) [inline intake/static/intake/scss/atoms/_layout.scss]: Don't need the html element in the selector probe

File: intake/static/intake/scss/atoms/_utilities.scss [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+.with-padding-large {
+  margin-bottom: $padding-large !important;
+}
+
+.with-padding-huge {
+  margin-bottom: $padding-huge !important;
+}
+
+.with-non-mobile-top-padding-small {
```
Comment (norrishung) [inline intake/static/intake/scss/atoms/_utilities.scss:112]: These classes might just be padding on a header organism if that's all these are for.

File: intake/static/intake/scss/templates/_main-landing.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+body.main-landing {
```
Comment (norrishung) [inline intake/static/intake/scss/templates/_main-landing.scss]: stay away from element tagnames in css if possible!

**Review by norrishung — COMMENTED** [review]
Everything looks pretty good! Just a couple of small tweaks but not super important.

**Review by jennifermarie — APPROVED** [review]

**Pull request merged**